src/components/playbackSettings/playbackSettings.template.html

```diff
@@ -157,6 +157,17 @@ <h2 class="sectionTitle">
         <div class="selectContainer">
             <select is="emby-select" class="selectSkipBackLength" label="${LabelSkipBackLength}"></select>
         </div>
+        <div class="selectContainer">
+            <select is="emby-select" id="selectPreferredTranscodeAudioCodec" label="${LabelSelectPreferredTranscodeAudioCodec}">
+                <option value="">${Auto}</option>
+                <option value="aac">AAC</option>
+                <option value="ac3">AC3</option>
+                <option value="dts">DTS</option>
+                <option value="eac3">EAC3</option>
```

Comment:
Just a note: EAC3 encoding is not enabled on server side yet, if this is ever merged before the server-side change, then this option has to be removed.

Reply:
This is just a hint. It has no effect if the codec is missing/disallowed on the  server. Probably should add a description to the field about this.